cobigen/cobigen-core-api/src/main/java/com/devonfw/cobigen/api/exception/RESTSearchResponseException.java

```diff
@@ -0,0 +1,40 @@
+package com.devonfw.cobigen.api.exception;
+
+/** Exception to indicate that the REST search API encountered a problem while accessing the server. */
+public class RESTSearchResponseException extends CobiGenRuntimeException {
+
+  private static final long serialVersionUID = 1L;
+
+  /**
+   * Creates a new {@link RESTSearchResponseException} with the given message
+   *
+   * @param message error message of the exception
+   */
+  public RESTSearchResponseException(String message) {
```

Comment:
No proper PascalCase, Please change to `RestSearchResponseException`

Reply:
Renamed.